### ISSUE TITLE
Typed-decision model interface: one plugin, pluggable backends (laya / Jev / local-logits)

### ISSUE THREAD

**Issue by OpakAlex** [opentalon/opentalon#361]
## Summary

The new `decide` block (opentalon/tln-language#205) has a **model mode** that delegates to an external "System 1" decision model via the existing `ToolResolver`. Rather than build a laya adapter and a Jev adapter separately, define **one general interface** — a typed-decision tool contract — that laya, Jev, and the small-LLM-logits recipe all implement as interchangeable backends.

## The interface

`decide … using model "m"` calls the resolver as:

```
Tools.Call(ctx, server="m", tool="decide", args={ state: string, choices: []string })
  → { chosen: string, confidence: float, probabilities: map[string]float }
```

That's the whole contract. Any decision model that answers a `decide` tool call with this shape is a valid backend — bound to a name with a `connector`:

```tln
connector "decider" via mcp {
  endpoint env "DECIDER_URL"
  model "..."
}

decide "email_kind" {
  for records where folder == "Inbox"
  choices ["Legitimate", "Spam", "Phishing"]
  ask concat("Subject: ", attr "subject", "\n\n", attr "body")
  using model "decider"
  confidence >= 0.9
}
```

## Proposed work: one plugin, pluggable backends

Ship a single tln plugin (e.g. `tln-decide`, sibling of `tln-mcp` / `tln-io`) exposing the `decide` tool, with a backend selector. Reference backends to implement:

- [ ] **laya** — [`convaiinnovations/laya-typed-decisions`](https://huggingface.co/convaiinnovations/laya-typed-decisions) (open-weights ModernBERT typed decisions: choice / score / noul)
- [ ] **Jev** — [typesafe.ai](https://typesafe.ai/) hosted System-1 API
- [ ] **local-logits** — any small GGUF LLM (e.g. Qwen3-0.6B), reading logits over the choice tokens and softmaxing, per [Jev in 25 lines](https://www.nobodywho.ai/posts/jev-in-25-lines/)

All three return the same `{chosen, confidence, probabilities}` shape, so they're swappable via config with no change to the `.tln` source.

## Notes

- Calibration matters: backends should return honest probabilities (the `confidence >=` gate depends on it), not overconfident argmax.
- The plugin should be documented tln-first (usage in `.tln`, not Go).
- Contract reference: [`docs/decide.md`](https://github.com/opentalon/tln-language/blob/master/docs/decide.md), [ADR-0014](https://github.com/opentalon/tln-language/blob/master/docs/design/0014-decide-typed-decisions.md).

Depends on opentalon/tln-language#205.
